packages/account/src/Sections/Profile/PhoneVerification/__test__/confirm-phone-number.spec.tsx

```diff
@@ -40,34 +57,88 @@ describe('ConfirmPhoneNumber', () => {
         expect(screen.getByRole('button', { name: 'Get code via SMS' })).toBeInTheDocument();
         expect(screen.getByRole('button', { name: 'Get code via WhatsApp' })).toBeInTheDocument();
     });
-    it('should call requestOnWhatsApp when Whatsapp button is clicked', () => {
+
+    it('should render validatePhoneNumber function on user key pressed', () => {
```

Comment:
1. Do we show error msg when field is empty? 
2. Do we also test for the case when user inputs some non-numerical value and input shld not accept/show it?
3. Is there any error when users types more than a certain amount of digits for phone number? 🙏

Reply:
@utkarsha-deriv the error message is returned based on the hooks, even though we type in something which is incorrect, it will return based on given error message. We already make the validation check in useRequestPhoneNumberOTP.spec.tsx

Reply:
I've moved the validation to a constant file, implemented the test case for phone number typed 🙏